_posts/2018-10-30-conducting-good-retrospectives.md

```diff
@@ -0,0 +1,110 @@
+
+#### Conversational Turn Taking
+
+It's very important that everyone can be heard. Your team should decide on a signal that makes it clear you have something to say next. My team has opted to use a finger raise when you want to say something. Once the person currently speaking is done, they or the facilitator would motion to the next person in line to speak. 
+
+Often one might have something to say that is tangential to the current conversation, and will delay their turn to speak to let the current thread of discussion finish.
+
+The primary motivation for the technique to is **avoid interruptions** and **talking over each other**.
+
+#### Format/Activity
+
+Free form conversation is not conducive to reaching conclusions, so it helps to pick a meeting format that fits well with your team. At the beginning of the meeting, the facilitator will explain the format and confirm that everyone is okay with that decision. Different formats optimize for different aspects of your team: project performance, interpersonal communication, or general frustration.
+
+Picking a format gives the **meeting structure** and often an **objective**, allowing the team to work together towards a known and common goal.
+
+#### Action Items
+
+A necessary output of the meeting is a list of action items. An action item is a task that requires further action in order to accomplish it and is assigned to a specific person. You've probably experienced the situation where everyone agrees something should get done, but no one ever does it.
+
+Sometimes you run into an action item that isn't a specific task, but possibly a general change in behavior that is meant to be actioned by the whole team. In this case, the real action item would be bearing the responsibility of shepherding the adoption of this new behavior for a couple weeks.
+
+Leaving the meeting with a list of action items ensures that the tasks you spent an hour finding, **actually get done**.
+
+## Case Study
+
+For the last year, my team has had a weekly retro, alternating the format each week. We operate in two week sprints, so in the middle of the sprint we typically concentrate on interpersonal issues or other general frustrations and at the end of the sprint we focus on project-specific issues.
+
+However, at the beginning of any retrospective, we solicit the opportunity to suggest an alternative format.
+
+#### Mid-Sprint Retrospective
+
+During the Mid-Sprint Retrospective, we engage in the [Lean Coffee](http://leancoffee.org) activity (unless someone suggests an alternate activity!). The chosen facilitator will explain the chosen activity and pass out the required supplies. 
+
+The basics of the Lean Coffee activity are:
+
+1. Generate discussion topics
+    - Limit to 5 minutes
+1. Vote on topics
+1. Discuss topics
+    - Limit to 5 minutes, then vote on continuing with a 2 minute extension, or else move on to next topic.
```

Comment:
Is it worth noting that we tinkered with the 5/2 durations - or does that just muddy the waters?